src/core/core/task.cpp

```diff
@@ -55,6 +55,7 @@
 namespace dsn 
 {
 __thread struct __tls_dsn__ tls_dsn;
+__thread uint16_t tls_dsn_lower32_task_id_mask = 0;
```

Comment:
Should not put in tls_dsn because tls_dsn will be memset as 0 every called by set_tls_dsn_context().
I add tls_dsn_lower32_task_id_mask here to avoid task id conflict when multiple set_tls_dsn_context(node, nullptr, nullptr) is invoked in one thread.